### ISSUE TITLE
Client crash when placing any building related blocks

### ISSUE THREAD

**Issue by DustRaven** [Direwolf20-MC/BuildingGadgets#575]
When I tried to place construction block powder into water with the building gadget, the game crashes, producing the following log: https://hastebin.com/ovedirojef.rb

Upon logging in again, everything is fine, until I tried to paste something (Green enchanted leaves from Oh the Biomes you'll go and crimson stem from minecraft), the game crashes again, producing almost the same log: https://hastebin.com/golonetivu.rb

I'm using some texture packs (128x from BDCraft) and the modpack "All the mods 6". My computer has 32G of RAM of which 8 are assigned to Minecraft.

* Forge version: 35.1.13
* Building Gadgets version: 3.7.3
* Crash 1: https://hastebin.com/ovedirojef.rb
* Crash 2: https://hastebin.com/golonetivu.rb

**Comment by MichaelHillcox** [comment]
Can you explain more about how and what blocks you are using as you shouldn't be getting any crashes with the powder. I've check this one out before.

**Comment by DustRaven** [comment]
Sure :) First I tried to place Construction Block Powder (buildinggadget:construction_block_powder) in the Water with the Building Gadget. Then I tried to paste the leaves (byg:green_enchanted_leaves) and the logs (minecraft:crimson_stem). 

Lately, my game is also crashing when I try to harvest dense construction blocks which were left over from earlier, but I don't have a crash log of this... Interestingly enough, my wife has the same mods and resource packs but it works for her.

Maybe it is something hardware related?
